README.md

```diff
@@ -91,6 +92,32 @@ is unset (those tests are not run). To run all tests, execute
 TEST_WITH_NETWORK=1 make test
 ```
 
+## Docker
+
```

Comment:
I suggest that the following is added to make it obvious:

````
Clone the Git reposistory and prepare the clone:
```sh
git clone https://github.com/zonemaster/zonemaster-ldns
cd zonemaster-ldns
perl Makefile.PL
```
````